src/shaders/globe_raster.fragment.glsl

```diff
@@ -1,9 +1,40 @@
 uniform sampler2D u_image0;
 varying vec2 v_pos0;
 
+#ifndef FOG
+uniform vec3 u_frustum_tl;
+uniform vec3 u_frustum_tr;
+uniform vec3 u_frustum_br;
+uniform vec3 u_frustum_bl;
+uniform highp vec3 u_globe_pos;
+uniform highp float u_globe_radius;
+uniform vec2 u_viewport;
+#endif
+
 void main() {
-    vec4 color = texture2D(u_image0, v_pos0);
+#if defined(CUSTOM_ANTIALIASING) && defined(USE_STANDARD_DERIVATIVES) && !defined(GLOBE_POLES)
```

Comment:
Thinking back on this, we can simplify to:
```suggestion
#if defined(CUSTOM_ANTIALIASING)
```
For two reasons:

1. we inject `CUSTOM_ANTIALIASING` only when the extension is available for use, otherwise it's redundant.
2. we should enable this on the poles, otherwise there are some situations where we would miss on the AA:

<img width="281" alt="Screen Shot 2022-05-03 at 8 54 57 AM" src="https://user-images.githubusercontent.com/7061573/166494158-19d7df34-edb4-4d59-93b8-9b39931ce0d7.png">

Here I'm putting the pixels affected by AA to white, just to visualize it ^

Reply:
Shouldn't we still have the check for standard derivatives though?

Reply:
Oh I see the first point got it!